management/models.py

```diff
@@ -1,14 +1,24 @@
-from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.models import AbstractUser, Group
 from django.db import models
-from guardian.shortcuts import assign_perm
+from guardian.shortcuts import assign_perm, get_anonymous_user
+
+from djangomain.settings import ANONYMOUS_USER_NAME
```

Comment:
Get the settings from `django.conf`, so as a more robust approach to using the settings. [More info](https://docs.djangoproject.com/en/5.0/topics/settings/#using-settings-in-python-code).
```suggestion
from django.conf import settings
```